docs: split up docs templates into different pages (#21574)
gatsbyjs/gatsby: docs/contributing/submit-to-plugin-library.md

1.  Publish a package to npm (learn how on the [npm docs](https://docs.npmjs.com/getting-started/publishing-npm-packages)).
2.  Include the [required files](/docs/files-gatsby-looks-for-in-a-plugin/) in your plugin code.
3.  **Include a `keywords` field** in your plugin's `package.json`, containing `gatsby` and `gatsby-plugin`. If your plugin is a theme, please also include `gatsby-theme`.

> 4.  Document your plugin with a README, there is a [plugin template](/contributing/how-to-write-a-plugin-readme/#near-perfect-example-of-a-plugin-readme) available to be used as reference.

Reviewer: With a little rewording, the whole plugin README contributing doc should suffice here.

```suggestion
4.  Document your plugin with a README, using the contributing [plugin template](/contributing/how-to-write-a-plugin-readme/) for reference.
```
